[2025-09-08 16:58:33,255][INFO] 执行 pytest 命令：python -m pytest -sv D:\PyCharmProjects\ums_uiautomator\test_suite\tmp_project.py --device_id=AF8YVB1805003480 --task_id=17 --alluredir=D:\PyCharmProjects\ums_uiautomator\result\17\allure_raw
[2025-09-08 16:58:37,426][INFO] pytest 输出（编码：ascii）：============================= test session starts =============================
[2025-09-08 16:58:37,427][INFO] pytest 输出（编码：ascii）：platform win32 -- Python 3.9.13, pytest-7.1.2, pluggy-1.6.0 -- C:\ProgramData\Anaconda3\python.exe
[2025-09-08 16:58:37,431][INFO] pytest 输出（编码：ascii）：cachedir: .pytest_cache
[2025-09-08 16:58:37,432][INFO] pytest 输出（编码：ascii）：metadata: {'Python': '3.9.13', 'Platform': 'Windows-10-10.0.26100-SP0', 'Packages': {'pytest': '7.1.2', 'py': '1.11.0', 'pluggy': '1.6.0'}, 'Plugins': {'allure-pytest': '2.9.45', 'anyio': '1.3.1', 'langsmith': '0.4.20', 'html': '3.1.1', 'metadata': '2.0.1'}, 'JAVA_HOME': 'C:\\Program Files\\Java\\jdk1.8.0_221'}
[2025-09-08 16:58:37,437][INFO] pytest 输出（编码：ascii）：rootdir: D:\PyCharmProjects\ums_uiautomator
[2025-09-08 16:58:37,437][INFO] pytest 输出（编码：ascii）：plugins: allure-pytest-2.9.45, anyio-1.3.1, langsmith-0.4.20, html-3.1.1, metadata-2.0.1
[2025-09-08 16:58:37,438][INFO] pytest 输出（编码：ascii）：collecting ... collected 4 items
[2025-09-08 16:58:37,439][INFO] pytest 输出（编码：ascii）：
[2025-09-08 16:58:37,815][INFO] pytest 输出（编码：ascii）：test_suite/tmp_project.py::test_case01 ERROR
[2025-09-08 16:58:37,817][INFO] pytest 输出（编码：ascii）：test_suite/tmp_project.py::test_case02 SKIPPED (Skip if demo)
[2025-09-08 16:58:37,846][INFO] pytest 输出（编码：ascii）：test_suite/tmp_project.py::test_case03 ERROR
[2025-09-08 16:58:37,879][INFO] pytest 输出（编码：ascii）：test_suite/tmp_project.py::test_case04 ERROR
[2025-09-08 16:58:37,880][INFO] pytest 输出（编码：ascii）：
[2025-09-08 16:58:37,880][INFO] pytest 输出（编码：ascii）：=================================== ERRORS ====================================
[2025-09-08 16:58:37,881][INFO] pytest 输出（编码：ascii）：________________________ ERROR at setup of test_case01 ________________________
[2025-09-08 16:58:37,881][INFO] pytest 输出（编码：ascii）：
[2025-09-08 16:58:37,882][INFO] pytest 输出（编码：ascii）：device_id = 'AF8YVB1805003480'
[2025-09-08 16:58:37,882][INFO] pytest 输出（编码：ascii）：
[2025-09-08 16:58:37,882][INFO] pytest 输出（编码：ascii）：@pytest.fixture(scope="function")
[2025-09-08 16:58:37,884][INFO] pytest 输出（编码：ascii）：def setup_and_teardown_demo(device_id):
[2025-09-08 16:58:37,884][INFO] pytest 输出（编码：ascii）："""
[2025-09-08 16:58:37,897][INFO] pytest 输出（编码：GB2312）：通用收尾：回到主页面
[2025-09-08 16:58:37,900][INFO] pytest 输出（编码：GB2312）：d 为 UI 测试的核心对象，除非知道后果否则请勿修改
[2025-09-08 16:58:37,901][INFO] pytest 输出（编码：ascii）：@Author: CN-LanBao
[2025-09-08 16:58:37,901][INFO] pytest 输出（编码：ascii）：@Create: 2022/5/9 17:32
[2025-09-08 16:58:37,902][INFO] pytest 输出（编码：ascii）：:return: Uiautomator
[2025-09-08 16:58:37,902][INFO] pytest 输出（编码：ascii）："""
[2025-09-08 16:58:37,902][INFO] pytest 输出（编码：ascii）：>       d.screen_on()
[2025-09-08 16:58:37,904][INFO] pytest 输出（编码：ascii）：E       AttributeError: 'NoneType' object has no attribute 'screen_on'
[2025-09-08 16:58:37,904][INFO] pytest 输出（编码：ascii）：
[2025-09-08 16:58:37,905][INFO] pytest 输出（编码：ascii）：test_suite\conftest.py:51: AttributeError
[2025-09-08 16:58:37,905][INFO] pytest 输出（编码：ascii）：________________________ ERROR at setup of test_case03 ________________________
[2025-09-08 16:58:37,906][INFO] pytest 输出（编码：ascii）：
[2025-09-08 16:58:37,906][INFO] pytest 输出（编码：ascii）：device_id = 'AF8YVB1805003480'
[2025-09-08 16:58:37,906][INFO] pytest 输出（编码：ascii）：
[2025-09-08 16:58:37,907][INFO] pytest 输出（编码：ascii）：@pytest.fixture(scope="function")
[2025-09-08 16:58:37,907][INFO] pytest 输出（编码：ascii）：def setup_and_teardown_demo(device_id):
[2025-09-08 16:58:37,907][INFO] pytest 输出（编码：ascii）："""
[2025-09-08 16:58:37,914][INFO] pytest 输出（编码：GB2312）：通用收尾：回到主页面
[2025-09-08 16:58:37,916][INFO] pytest 输出（编码：GB2312）：d 为 UI 测试的核心对象，除非知道后果否则请勿修改
[2025-09-08 16:58:37,916][INFO] pytest 输出（编码：ascii）：@Author: CN-LanBao
[2025-09-08 16:58:37,917][INFO] pytest 输出（编码：ascii）：@Create: 2022/5/9 17:32
[2025-09-08 16:58:37,917][INFO] pytest 输出（编码：ascii）：:return: Uiautomator
[2025-09-08 16:58:37,918][INFO] pytest 输出（编码：ascii）："""
[2025-09-08 16:58:37,918][INFO] pytest 输出（编码：ascii）：>       d.screen_on()
[2025-09-08 16:58:37,918][INFO] pytest 输出（编码：ascii）：E       AttributeError: 'NoneType' object has no attribute 'screen_on'
[2025-09-08 16:58:37,919][INFO] pytest 输出（编码：ascii）：
[2025-09-08 16:58:37,919][INFO] pytest 输出（编码：ascii）：test_suite\conftest.py:51: AttributeError
[2025-09-08 16:58:37,920][INFO] pytest 输出（编码：ascii）：________________________ ERROR at setup of test_case04 ________________________
[2025-09-08 16:58:37,920][INFO] pytest 输出（编码：ascii）：
[2025-09-08 16:58:37,921][INFO] pytest 输出（编码：ascii）：device_id = 'AF8YVB1805003480'
[2025-09-08 16:58:37,921][INFO] pytest 输出（编码：ascii）：
[2025-09-08 16:58:37,922][INFO] pytest 输出（编码：ascii）：@pytest.fixture(scope="function")
[2025-09-08 16:58:37,922][INFO] pytest 输出（编码：ascii）：def setup_and_teardown_demo(device_id):
[2025-09-08 16:58:37,923][INFO] pytest 输出（编码：ascii）："""
[2025-09-08 16:58:37,928][INFO] pytest 输出（编码：GB2312）：通用收尾：回到主页面
[2025-09-08 16:58:37,930][INFO] pytest 输出（编码：GB2312）：d 为 UI 测试的核心对象，除非知道后果否则请勿修改
[2025-09-08 16:58:37,931][INFO] pytest 输出（编码：ascii）：@Author: CN-LanBao
[2025-09-08 16:58:37,931][INFO] pytest 输出（编码：ascii）：@Create: 2022/5/9 17:32
[2025-09-08 16:58:37,932][INFO] pytest 输出（编码：ascii）：:return: Uiautomator
[2025-09-08 16:58:37,933][INFO] pytest 输出（编码：ascii）："""
[2025-09-08 16:58:37,933][INFO] pytest 输出（编码：ascii）：>       d.screen_on()
[2025-09-08 16:58:37,934][INFO] pytest 输出（编码：ascii）：E       AttributeError: 'NoneType' object has no attribute 'screen_on'
[2025-09-08 16:58:37,934][INFO] pytest 输出（编码：ascii）：
[2025-09-08 16:58:37,935][INFO] pytest 输出（编码：ascii）：test_suite\conftest.py:51: AttributeError
[2025-09-08 16:58:37,935][INFO] pytest 输出（编码：ascii）：=========================== short test summary info ===========================
[2025-09-08 16:58:37,936][INFO] pytest 输出（编码：ascii）：ERROR test_suite/tmp_project.py::test_case01 - AttributeError: 'NoneType' obj...
[2025-09-08 16:58:37,936][INFO] pytest 输出（编码：ascii）：ERROR test_suite/tmp_project.py::test_case03 - AttributeError: 'NoneType' obj...
[2025-09-08 16:58:37,937][INFO] pytest 输出（编码：ascii）：ERROR test_suite/tmp_project.py::test_case04 - AttributeError: 'NoneType' obj...
[2025-09-08 16:58:37,937][INFO] pytest 输出（编码：ascii）：======================== 1 skipped, 3 errors in 0.45s =========================
